app/components/advice_component.html.erb

```diff
@@ -0,0 +1,6 @@
+<aside class="app-advice">
```

Comment:
I can see that you have set the base branch correctly on Github, but have you rebased from `2937-add-content-for-quals-to-course-detail-pages`?

The component files files have been added there, and so shouldn't show on this PR. Nothing urgent, but could possibly cause a merge conflict down the road. 😄

Reply:
Oop i think you have rebased but maybe before I pushed up the last commit where I added the component! 😓